README.md

```diff
@@ -1 +1,23 @@
-# WeatherForcast-APP
\ No newline at end of file
+# WeatherForcast-APP
```

Comment:
Can you add how to start/use the app on the README both in development mode and also add a live link?

Reply:
but surprisingly everything works well on my machine, the problem is with the repository I will create a new repo and upload again

Reply:
good

Reply:
I will contact you via slack.
I created a new repo and tried to upload the same files and the dis folder and  node_modules didn't get uploaded to the repo. that where the problems cause on this repo I had to manually upload the dist folder

Reply:
normally, you shouldn't upload the `node_modules/` folder, it should be mentioned in the `.gitignore` file